tests/controllers/users.test.js

```diff
@@ -29,8 +29,37 @@ tape('test /users GET returns list of users', t => {
           if (err) t.fail(err)
           // check our get path returns that user correctly
           t.equal(res.body.length, 2, 'response body should be an array with length 1')
-          t.ok(res.body.map((user) => user.username).includes(validUser1.username), 'mattlub has been added')
-          t.ok(res.body.map((user) => user.username).includes(validUser2.username), 'm4v15 has been added')
+          t.ok(res.body.map(user => user.username).includes(validUser1.username), 'mattlub has been added')
+          t.ok(res.body.map(user => user.username).includes(validUser2.username), 'm4v15 has been added')
+          dropCollectionAndEnd(User, t)
+        })
+    })
+    .catch(err => t.end(err))
+})
+
+// GET /users/:id
+tape('test /users/:id GET with id of something not in the database', (t) => {
+  supertest(server)
+    .get('/users/10')
+    .expect(404)
+    .expect('Content-Type', /json/)
+    .end((err, res) => {
+      if (err) t.fail(err)
+      t.equal(res.body.message, 'Cannot find user with id=10', 'response message should be "Cannot find user with id=10"')
+      dropCollectionAndEnd(User, t)
+    })
+})
+
+tape('test /users/:id GET with id of something in the database', (t) => {
```

Comment:
shall we change the req parameter to _id so ```/users/:_id``` as this is how we need it to look for mongoose
@mattlub